### DISCUSSION TITLE
Unable to Find Pipenv Virtual Environment in VSCode Jupyter Notebook

### DISCUSSION THREAD

**Discussion by LiPingYen** [microsoft/vscode-jupyter#15564] [Questions and Answers]
I'm encountering an issue while using Jupyter Notebook within VSCode where I can't locate the Python virtual environment created by `pipenv`.

When I run a Python script, the pipenv virtual environment is found without any problem. Currently, it seems that when selecting the Jupyter kernel, VSCode can successfully locate the base and conda-related virtual environments, but it fails to find the one created by `pipenv`. 

I've searched for relevant articles but haven't found a suitable solution. Has anyone experienced a similar situation and found a resolution? Any insights or solutions would be greatly appreciated.

**Comment by DonJayamanne** [comment]
Do you see the pipenv environment when using the command `Python: Select Interpereter`

**Comment by LiPingYen** [comment]
Hi @DonJayamanne,

Yes, the pipenv environment is shown when I search in `Python: Select Interpereter`.

1. Python environment in `Python: Select Interpereter`:
<img width="1063" alt="CleanShot 2024-04-18 at 22 34 44@2x" src="https://github.com/microsoft/vscode-jupyter/assets/47491041/bd520d04-5b44-46d9-bfa9-d641488f38e0">

2. Python environment in `Jupyter: Select Intepreter to start jupyter notebook server`:

![CleanShot 2024-04-18 at 22 34 22@2x](https://github.com/microsoft/vscode-jupyter/assets/47491041/126bf928-499b-408e-b470-7e422dbe567a)

**Comment by DonJayamanne** [comment]
Closing as a duplicate of https://github.com/microsoft/vscode-python/issues/23260
